src/ale_interface.cpp

```diff
@@ -147,25 +147,25 @@ float ALEInterface::getFloat(const std::string& key) {
 }
 
 // Set the value of a setting.
-void ALEInterface::set(const string& key, const string& value) {
+void ALEInterface::setString(const string& key, const string& value) {
   assert(theSettings.get());
   assert(theOSystem.get());
   theSettings->setString(key, value);
   theSettings->validate();
 }
-void ALEInterface::set(const string& key, const int& value) {
+void ALEInterface::setInt(const string& key, const int& value) {
```

Comment:
Unless there's a reason, I would pass the ints, bools, and floats by value rather than reference. This lets you call setInt() with a temporary, e.g. the return value of a variable, which you can't do otherwise.